protocol/storage/miner_test.go

```diff
@@ -99,12 +82,15 @@ func TestReceiveStorageProposal(t *testing.T) {
 		_, err := miner.receiveStorageProposal(context.Background(), proposal)
 		require.NoError(t, err)
 
-		assert.True(t, accepted, "Proposal has been accepted")
-		assert.False(t, rejected, "Proposal has not been rejected")
-		assert.Equal(t, "", message)
+		// one deal should be stored and it should have been accepted
+		require.Len(t, porcelainAPI.deals, 1)
+		for _, deal := range porcelainAPI.deals {
+			assert.Equal(t, storagedeal.Accepted, deal.Response.State)
```

Comment:
should there be a signature check here too or is this covered by the other tests?

Reply:
This comment convinced me that the signing of accepted deals should have it's own test, so I took it out of the first test and added a new test to check signing specifically. I also refactored a bit to reduce the test setup in these tests.